files/grest/rpc/assets/asset_info_bulk.sql

```diff
@@ -34,7 +34,7 @@ BEGIN
     SELECT
       ENCODE(ma.policy, 'hex'),
       ENCODE(ma.name, 'hex'),
-      ENCODE(ma.name, 'escape'),
+      ENCODE(grest.cip68_strip_label(ENCODE(ma.name, 'hex')), 'escape'),
```

Comment:
was just wondering post implementatin, strictly speaking - wouldnt this be cip67 and not 68?

Reply:
You are correct, I initially called it cip67 but most just talk about cip68 so wasn't sure if it caused confusion to mix. Only internal anyway but happy to change to cip67